base.js

```diff
@@ -30,5 +30,10 @@ module.exports = {
     'prefer-template': 'error',
     'prefer-object-spread': 'error',
     'comma-dangle': 'off',
+    camelcase: 'error',
```

Comment:
What happens if we want to create a constant, like `const MAX_TIMEOUT = 100;`? Will it reject this?

Reply:
![image](https://user-images.githubusercontent.com/31747499/145384763-fc08687a-a34e-49de-8aa6-950ee2f2f7d3.png)
It wouldn't, I tested this point.